core/org.osate.aadl2/src/org/osate/aadl2/util/Aadl2Util.java

```diff
@@ -154,9 +154,9 @@ public static String getPrintableSOMMembers(SystemOperationMode som) {
 		}
 		for (ModeInstance mode : som.getCurrentModes()) {
 			if (members.isEmpty()) {
-				members = mode.getComponentInstancePath();
+				members = "\"" + mode.getComponentInstancePath() + "\"";
 			} else {
-				members = members + ", " + mode.getComponentInstancePath();
+				members = members + ", \"" + mode.getComponentInstancePath() + "\"";
```

Comment:
This encloses each individual mode name in quotes not the list as a whole. Also, the quotes only need to be added in the CSV report.